### ISSUE TITLE
(bug) Download not working

### ISSUE THREAD

**Issue by MiroKopecky** [DisboxApp/web#35]
Hi, I can't download any files. Attaching screenshot with error and dev console.

(after click on download)
![image](https://user-images.githubusercontent.com/53955446/226107195-91f2deca-b3fe-4a6e-a68c-fc8fd59ef284.png)

(after click on OK in alert)
![image](https://user-images.githubusercontent.com/53955446/226107227-2b8f1416-f954-428e-a3c1-2a86dc045e29.png)

**Comment by DisboxApp** [comment]
Hey, this is probably caused because of your Chrome version, what version are you using? Try updating chrome to the latest version

**Comment by MiroKopecky** [comment]
Gey, I'm using Brave browser which is up to date
![image](https://user-images.githubusercontent.com/53955446/226125061-f98c30b5-598e-444a-bc3b-9e482a52bc0f.png)

gonna try Chrome and Edge now

**Comment by MiroKopecky** [comment]
so on Edge it's working, what could be the cause of the problem in Brave browser ?

**Comment by DisboxApp** [comment]
Yeah.
Disbox uses an the experimental File System Access API for saving files, Brave is based on Chrome, so it should have it, but from brave/brave-browser#20563 and brave/brave-browser#20563 it looks like it's disabled by default for security measures.

There's a guide on both the issues I linked on how to enable this feature. Glad we found a solution!

This does get me wondering if using this experimental API was even a good idea. I looked for a better solution for a while and couldn't find anything with all the features I wanted except the File System Access API, but it sucks that it hasn't been adopted (or enabled by default) by more browsers by now.
